xl/formatter.py

```diff
@@ -410,7 +410,8 @@ def format(self, track, markup_escape=False):
 
         return Formatter.format(self)
 
-class TagFormatter():
+# XXX: should inherite from class Formatter?
```

Comment:
This should probably be clarified before merging. Otherwise the code looks good to me.